gui/static/js/components/controlpanel.js

```diff
@@ -166,6 +166,68 @@ export default class ControlPanel extends React.Component {
      * Handles onChange for bot dropdown. Changes currently selected bot.
      */
     selectBot(event) {
+        this.setState({currentBot: event.target.value});
+        // Insert code to change Blockly toolbox here.
```

Comment:
If you are planning on removing this code, please delete. If you are commenting it out right now because you are planning on testing this later, it would be better if you copy pasted this into another branch for another pull request.